libs/examples/src/index.ts

```diff
@@ -115,7 +115,7 @@ import { DtExampleDrawerNested } from './drawer/drawer-nested-example/drawer-nes
 import { DtExampleDrawerOver } from './drawer/drawer-over-example/drawer-over-example';
 import {
   DtExampleCustomEmptyStateTable,
-  DtExampleCustomEmptyState,
+  DtExampleCustomEmptyState
```

Comment:
Same here, should the linter not complain about trailing commas? can we please check?